src/modules/global-messages/global-messages.tsx

```diff
@@ -0,0 +1,53 @@
+import { MessageBox } from '@atb/components/message-box';
+import { getTextForLanguage, useTranslation } from '@atb/translations';
+import { and } from '@atb/utils/css';
+import { useActiveGlobalMessages } from './context';
+import { GlobalMessageContextEnum } from './types';
+import style from './global-messages.module.css';
+import { motion } from 'framer-motion';
+
+export type GlobalMessagesProps = {
+  context: GlobalMessageContextEnum;
+  className?: string;
+};
+
+export function GlobalMessages({ context, className }: GlobalMessagesProps) {
+  const { language } = useTranslation();
+  const { activeGlobalMessages } = useActiveGlobalMessages();
+
+  if (!activeGlobalMessages || activeGlobalMessages.length === 0) return null;
```

Comment:
I don't really know if i prefer this or if it is clearer or more unclear 🤔 Just thinking out loud

```suggestion
  if (!activeGlobalMessages?.length) return null;
```

Reply:
That being said. Why would `activeGlobaleMessages` ever be `undefined`? I think that is a modeling fault. One of the biggest advantages of Arrays and Lists is that they can contain the same API interfaces if they have no content. So there should be very few cases where lists are undefined, but rather be instantiated as empty

Reply:
`activeGlobalMessages` won't be `undefined` so this is an unnecessary check. I've updated and fixed this to `!activeGlobalMessages.length` 👍